packages/html-test-app/src/preview-examples/event-list-custom-item-height-in-number.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Event List Custom Item Height Dynamic List</title>
+    <script type="module">
+      document.addEventListener('DOMContentLoaded', () => {
```

Comment:
IMO this is way to complicated. Just add the list items directly to the body tag below.